src/helpers/util.js

```diff
@@ -281,7 +281,7 @@ class Util {
    * @param {Object|Array} config
    * @param {Object} projectName
    */
-  static printListAsTree(config, projectName) {
+  static printListAsTree(config, projectName, listLimit) {
```

Comment:
listLimit parameter is redundant here
```suggestion
  static printListAsTree(config, projectName) {
```